coops-ui/src/components/dialogs/WarningPopover.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-menu v-if="dialogObj != null"
+    :close-on-content-click="false"
+    :nudge-width="200"
+    offset-y
+    max-width="18rem"
+  >
+    <template v-slot:activator="{ on }">
+      <span class="warning-text" v-on="on">
+        <v-icon class="warning-icon">mdi-information-outline</v-icon>{{ dialogObj.title }}
+      </span>
+    </template>
+    <v-card>
+      <v-list-item>
+        <v-list-item-content>
+         {{ dialogObj.msg }}
+        </v-list-item-content>
+      </v-list-item>
+    </v-card>
+  </v-menu>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { AlertMessage } from '../../interfaces'
+
+@Component({})
+export default class WarningPopover extends Vue {
+  @Prop({ default: null })
+  private dialogObj: AlertMessage
+}
+</script>
+
+<style lang="scss">
+.warning-text {
+  font-weight: bold;
+  font-size: 14px;
+  color: #2196F3;
+    .v-icon {
+    vertical-align: bottom !important;
+    color:#2196F3 !important;
```

Comment:
Guideline: Use colour names when available (in this case, "Dodger Blue"). Also on line 38.

Ideal: Use CSS variables from a defined app theme (see theme.scss). This allows reuse as well as centralized app colours (ie, theme).

Recommendation: Add this colour to the theme file and use the CSS variable here. You'll need to add `@import '@/assets/styles/theme.scss';` at the top of this section.